docs/tutorials/standalone/Soil/layered_soil.jl

```diff
@@ -0,0 +1,161 @@
+# This shows how to run single colum soil model, in standalone mode
+# with spatially varying properties. We are mimicking the experiment
+# carried out in Huang et. al.
+# Can. J. Soil Sci. (2011) 91: 169183 doi:10.4141/CJSS09118,
+# which measured the infiltration of layered soil in Fort McMurray,
+# Alberta, Canada. We thank Mingbin Huang and S. Lee Barbour for
+# correspondence and support, including sharing of data, with us
+# Note that all data used in this tutorial is available in their
+# publication.
+
+using Plots
+import SciMLBase
+import ClimaTimeSteppers as CTS
+using ClimaCore
+import CLIMAParameters as CP
+using ArtifactWrappers
+using DelimitedFiles: readdlm
+
+using ClimaLand
+using ClimaLand.Domains: Column
+using ClimaLand.Soil
+import ClimaLand
+FT = Float64;
+
+# Define simulation times
+t0 = Float64(0)
+tf = Float64(60 * 60)
+dt = Float64(30);
+
+# Define the domain 
+zmax = FT(0)
+zmin = FT(-1.1)
+nelems = 75
+Δ = FT((zmax - zmin) / nelems / 2)
+soil_domain = Column(; zlim = (zmin, zmax), nelements = nelems);
+
+# Download the parameter data. This has been obtained from
+# Table 1b of
+# Infiltration and drainage processes in multi-layered coarse soils
+# Mingbin Huang et. al.
+# Can. J. Soil Sci. (2011) 91: 169183 doi:10.4141/CJSS09118
+af = ArtifactFile(
+    url = "https://caltech.box.com/shared/static/qvbt37xkwz8gveyi6tzzbs0e18trpcsq.csv",
+    filename = "sv_62.csv",
+)
+dataset = ArtifactWrapper(@__DIR__, "sv62", ArtifactFile[af]);
+dataset_path = get_data_folder(dataset);
+data = joinpath(dataset_path, af.filename)
+parameter_data = readdlm(data, ',');
+# Our model treats z as increasing in the upwards direction.
+# Values below the surface are negative.
+# Because of this, we convert the (positive-valued) depth
+# of the data into a monotonically increasing z coordinate value.
+# using a negative sign and the reverse function.
+depth = reverse(-parameter_data[1, :] .* 0.01) # convert to m
+ksat = reverse(parameter_data[6, :] .* 1 / 100.0 / 60.0) # convert cm/min to m/s
+vgα = reverse(parameter_data[4, :] .* 100 * 2) # they report αᵈ; αʷ = 2αᵈ. This experiment is for infiltration (wetting).
+vgn = reverse(parameter_data[5, :])
+residual_frac = reverse(parameter_data[2, :])
+porosity = reverse(parameter_data[3, :]);
+
+# Create fields corresponding to the parameter
+ν = SpaceVaryingInput(depth, porosity, soil_domain.space.subsurface)
+K_sat = SpaceVaryingInput(depth, ksat, soil_domain.space.subsurface)
+θ_r = SpaceVaryingInput(depth, residual_frac, soil_domain.space.subsurface)
+hcm = SpaceVaryingInput(
+    depth,
+    (; α = vgα, n = vgn),
+    soil_domain.space.subsurface,
+    vanGenuchten{FT},
+)
+params = ClimaLand.Soil.RichardsParameters(;
+    ν = ν,
+    hydrology_cm = hcm,
+    K_sat = K_sat,
+    S_s = ClimaCore.Fields.zeros(soil_domain.space.subsurface) .+ 1e-3,## FIX
```

Comment:
ah, we dont have data for this. so I am treating it as constant. But it's easier to just make the field directly, rather than go through the SpaceVaryingInput function.

The limitation Im working with (why it has to be a field rather than a scalar) is because the type of all parameters \nu, K_sat, S_s, \theta_r are the same in the parameter struct definition. so if some are fields, they all need to be.

I left some comments so that it is clearer why I am doing this. if there is a better way of handling the struct def to be more flexible im open to changing it